api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -216,4 +216,30 @@
 								 baseTableName="encounter_diagnosis" baseColumnNames="changed_by"
 								 referencedTableName="users" referencedColumnNames="user_id" />
 	</changeSet>
+	<changeSet id="TRUNK-4791-20180214-1" author="patrick">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="provider" columnName="role"/>
+			</not>
+		</preConditions>
+		<comment>Adding "role" column to provider table</comment>
+		<addColumn tableName="provider">
+			<column name="role" type="int">
+				<constraints nullable="true"/>
+			</column>
+		</addColumn>
+	</changeSet>
+	<changeSet id="TRUNK-4791-20180214-2" author="patrick">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="provider" columnName="speciality"/>
+			</not>
+		</preConditions>
+		<comment>Adding "speciality" column to provider table</comment>
+		<addColumn tableName="provider">
```

Comment:
If the role is gonna be a concept id, then you need to add a foreign key constraint for it

Reply:
Done